include/net/ethernet.h

```diff
@@ -329,7 +329,6 @@ struct ethernet_context {
  */
```

Comment:
The "net/ethernet: Remove as many ifdef for CONFIG_NET_VLAN as possible" could be sent to upstream separately.